### DISCUSSION TITLE
Continuous variables are treated by dummy variables

### DISCUSSION THREAD

**Discussion by XiangBu** [pycaret/pycaret#1822] [Q&A] [answered]
hi,I have specified the categorical variables in the setup, however the continuous variables (gcs) are treated as dummy variables in the subsequent modeling, how can I stop this?Thanks in advance!
![image](https://user-images.githubusercontent.com/46986985/140844093-99abdf60-3b13-4677-be61-c558638df140.png)

![image](https://user-images.githubusercontent.com/46986985/140844043-ea955b25-28a9-4067-be10-a961a21af479.png)

**Comment by moezali1** [comment, marked as answer]
@XiangBu You can force the dtypes in the `setup` function. In this case, pass `numeric_features = ['gcs']` in the `setup` function.

**Reply by XiangBu** [reply]
It works well! Thanks
